hyprctl/main.cpp

```diff
@@ -136,10 +134,12 @@ int main(int argc, char** argv) {
     else if (!strcmp(argv[1], "dispatch")) dispatchRequest(argc, argv);
     else if (!strcmp(argv[1], "keyword")) keywordRequest(argc, argv);
     else if (!strcmp(argv[1], "--batch")) batchRequest(argc, argv);
+    else if (!strcmp(argv[1], "--help")) printf(USAGE.c_str());
```

Comment:
`printf` requires a string literal as its first argument. You should do `printf("%s", USAGE.c_str())` like in line 139 instead.